### PR TITLE
Correct package structure from PR #1

### PR DESCRIPTION
**Pull request by ghost**
- Moved dir `./project/models -> ./gateway/models`
- Moved dir `./project/resources -> ./gateway/resources`
- Moved file `./project/__init__.py -> ./gateway/__init__.py`
- Moved file `./app.py -> ./gateway/app.py`
- Moved file `./db.py -> ./gateway/db.py`
- Correct `gateway` import paths for `./tests/conftest.py`
- Corrected import paths and then commented-out not-working tests in
  `./tests`:
  - `test_firebase_model.py`
  - `test_firebase_resource.py`
  - `test_models.py`
- Added 2 simple tests in `./tests/test_the_simplest_thing.py`
- Added *Contributing* section to README, from mblomdahl/qsck#1
- Updated `setup.py` with
  - correct spelling
  - target package `gateway`
  - `setup_requires` and `install_requires` attributes

### CODE REVIEW

File: tests/test_the_simplest_thing.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+def test_true_is_not_false():
+
+    assert True is not False
+
+
+def test_tori_is_sometimes_clever():
+
+    assert 'Tori is sometimes clever'
```
Comment (mblomdahl) [inline tests/test_the_simplest_thing.py]: 🤣 👍 

Very professional!

File: gateway/resources/firebase.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from flask_restful import Resource, reqparse
 from typing import Tuple
-from models.firebase import FirebaseModel
+from ..models.firebase import FirebaseModel
```
Comment (mblomdahl) [inline gateway/resources/firebase.py:3]: Purrfect. That's how you do it! Relative imports FTW.

File: README.md [hunk 3]
```diff
@@ -13,8 +13,22 @@ Create a Python 3.6 virtual environment and install the packages:
...
+
+
+Contributing
+------------
+
+Really? Very welcome. Do the usual fork-and-submit-PR thingy.
+
+Running the tests:
+
+    python setup.py test
```
Comment (mblomdahl) [inline README.md:33]: Have you tried `deactivate; rm -rf venv`, running the *Getting Started* steps from scratch, then this passes the 2 dead-simple tests recommended in PR #1 comment..? Can't try it out myself atm.

**Review by mblomdahl — COMMENTED** [review]
Awesome. Check it out mjao, @akebrissman!

**Review by akebrissman — APPROVED** [review]
I agree to all the changes and I am looking forward to see how we can get tests with database and flask interaction working.

File: tests/test_models.py [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-    THEN check the email, hashed_password, authenticated, and role fields are defined correctly
-    """
-    assert new_fb.imsi == '123456789012345'
-    assert new_fb.token == 'guid'
+# """
+# This file (test_models.py) contains the unit tests for the models.py file.
+# """
+#
+#
+# def test_new_user(new_fb):
```
Comment (mblomdahl) [inline tests/test_models.py]: A better approach here would be to add the [skip decorator](http://doc.pytest.org/en/latest/skipping.html#skipping-test-functions).

- [x] revertera utkommenterad kod
- [x] lägg till skip-dekoratorn
- [x] testkör och verifiera

File: tests/test_the_simplest_thing.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+def test_true_is_not_false():
```
Comment (mblomdahl) [inline tests/test_the_simplest_thing.py:8]: I would like a test case that actually imports something from the `gateway` library. If the package structure inside `./gateway` dir was still broken, chances are these would still succeed.

2 things to test:

- [x] import this SQLAlchemy database thingy and assert it's just that, i.e. `test_db_is_a_sqlalchemy_database` test case with a `assert isinstance(db, SQLAlchemy)`
- [x] import the *FirebaseId* class from `./gateway/resources/firebase.py` and have a test like `test_firebase_id_is_a_flask_restful_resource` (`assert issubclass(FirebaseId, flask_restful.Resource)`)

**Review by mblomdahl — CHANGES_REQUESTED** [review]

File: tests/test_the_simplest_thing.py (hunk 5 above)
Comment (ghost) [reply tests/test_the_simplest_thing.py:8]: Done!

File: tests/test_the_simplest_thing.py (hunk 1 above)
Comment (ghost) [reply tests/test_the_simplest_thing.py]: It is not professional Removed now.

File: tests/test_the_simplest_thing.py [hunk 6]
```diff
@@ -1,9 +1,20 @@
+import flask_restful
+
+from gateway import db
+from gateway import SQLAlchemy
```
Comment (mblomdahl) [inline tests/test_the_simplest_thing.py]: - [x] Import SQLAlchemy directly, not indirectly from gateway package

**Review by mblomdahl — CHANGES_REQUESTED** [review]

**Review by mblomdahl — APPROVED** [review]

**Comment by mblomdahl** [comment]
Good job @vtepliuk! 🌸 Pure Gold™️.

**Pull request merged**